redhat-developer/app-services-go-linter
# chore: bump version of go and tools

Make lint throws following error with go1.18, which needed version update in the custom linter.

```
appservicesgolinter: internal error: package "fmt" without types was imported from "github.com/redhat-developer/app-services-cli/pkg/core/errors"
```

### Verification steps

1. Build the binary with the command:
```
go build -o app-services-go-linter ./cmd/app-services-go-linter
```
2. Copy the generated binary `app-services-go-linter` to `app-services-cli` repo.
3. Run the binary file, and lint checks for i18n keys should be performed.
```
./app-services-go-linter -path ./pkg/core/localize/locales ./...
```
